components/Datepicker/react.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Calendar from 'react-modern-calendar-datepicker'
```

Comment:
This library seems to support two components, one called Calendar and one called DatePicker. DatePicker is the default one, so we're essentially importing their DatePicker but calling it Calendar, which is pretty confusing.